common/djangoapps/util/enterprise_helpers.py

```diff
@@ -120,3 +122,50 @@ def insert_enterprise_pipeline_elements(pipeline):
 
     for index, element in enumerate(additional_elements):
         pipeline.insert(insert_point + index, element)
+
+
+def get_enterprise_customer_logo_url(request):
+    """
+    Client API operation adapter/wrapper
+    """
+    logo_url = None
```

Comment:
please move logo_url closer to where it is being used -- at least below the guard clause.